NameGend.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Sep 10 14:07:15 2019
+@author: tickc
+"""
+
+
+import csv
+with open('C:/Users/tickc/OneDrive/Documents/GitHub/gender/data/country_pop.csv') as g:
```

Comment:
Lots of hardcoded file paths in here. Consider using the get_data pattern from the genderize class instead.